tests/MyGame/Example/Stat.cs

```diff
@@ -117,4 +117,19 @@ public class StatT
 }
 
 
+// Verification function for 'Stat' table.
+static public class StatVerify
+{
+  static public bool Verify(Google.FlatBuffers.Verifier verifier, uint tablePos)
+  {
+    bool result = true;
+    result = result && verifier.VerifyTableStart(tablePos);
```

Comment:
Can the be generated like return verfier.VerifyTableStart(tablePos) && verifier.VerifyString(...) ...;

Seems like short circuit logic would be a win.

Reply:
switched to short circuit logic